lib/tether/tether_linux.go

```diff
@@ -126,10 +137,16 @@ func (t *tether) childReaper() error {
 func (t *tether) stopReaper() {
```

Comment:
can you confirm that we don't call this until all directly launched children have exited?